homeassistant/components/homematic/__init__.py

```diff
@@ -528,8 +528,13 @@ def _get_devices(hass, discovery_type, keys, interface):
                 "%s: Handling %s: %s: %s", discovery_type, key, param, channels
             )
             for channel in channels:
+                channel_name = device.CHANNELS[channel].NAME
+                if key not in channel_name:
```

Comment:
Please add a comment to explain this a bit (short)

Reply:
Target is to handle name resolution of channels similar to devices. In case the key itself is already the `channel_name`, the procedure will be skipped. Otherwise, the assigned "friendly" `channel_name` will be used.

before:
![grafik](https://user-images.githubusercontent.com/4221212/97802032-fcd16c80-1c40-11eb-8d42-9816cd13ba28.png)

after:
![grafik](https://user-images.githubusercontent.com/4221212/97802176-e7107700-1c41-11eb-954f-4f5eef403ba6.png)